standard/info/commons/info.lua

```diff
@@ -31,6 +31,13 @@ return {
 			hasSpecialTeam = false,
 			allowManual = true,
 		},
+		transfers = {
+			showTeamName = false, -- bool?
```

Comment:
Would anyone oppose making it always true? I can't see a good reason to not display it

Reply:
imo display wise it looks better without it 
but showing it serves a good purpose regarding information for teams with default logo

Reply:
How much additional space is used when showing it?

Reply:
https://liquipedia.net/leagueoflegends/Player_Transfers

Reply:
Ah, i thought it's shown below, not besides the logo.
IMO it's fine as long as we don't flip it like on LOL.
Doesn't really make sense as it's not a "versus" situation

Reply:
fwiw the current (commons) module flips it
we can not flip it of course if we think that is the better way to go

Reply:
went ahead and kicked the `showTeamName` option and always applied it
if @iMarbot thinks it is a bad idea we can discuss it

@mbergen for the flip stuff: if you want that changed please make a new comment for it ;)

Reply:
as per the module sync meeting yesterday we think having it as an option is the better way
- shortname can be a mess (pretty long) on some wikis
- some wikis certainly wouldn't like it
- gets very messy for double transfer stuff (screenshot see below)

since currently only 11 of our wikis add the shortnames i made it the default to not show

if this gets approved we would have to adjust module:info on 11 wikis due to that (sep. PR) before rollout on those wikis

Reply:
Ok, but I think once we will have a design-pass over it, I suspect that shortnames will be made mandatory. Having just the team's icon is not friendly for users.